### PR TITLE
Feature: strip version prefix

### PR DESCRIPTION
**Pull request by boesing**
This relates to #23

### CODE REVIEW

**Comment by boesing** [comment]
Mhm, any suggestions on how to pass the slevomat coding style for PHP 7.4?
One could handle it as in laminas to only apply coding style rules in a specific PHP version.
Its quite annoying if there are PHP version requirements in codestandards which cannot get applied due to the limitation of `composer.json`.

**Comment by Slamdunk** [comment]
Add an environment variable for the lowest step, and only run phpcs if the variable is set, see for example https://github.com/laminas/laminas-validator/blob/master/.travis.yml#L22

Coding standard should run in the lowest supported language version only

**Comment by boesing** [comment]
Yah I know that but wasnt enabled here and I dont think that should be part of this PR. Maybe I create another PR on monday.

**Comment by Slamdunk** [comment]
It's a small library, just do it here, to me

**Comment by boesing** [comment]
@malukenho tell me 🤷🏼‍♂️
Im fine with both 

- 2nd PR
- in here 

And: are you fine with this Feature.

File: test/BumpIntoTest.php [hunk 1]
```diff
@@ -154,5 +153,32 @@ public function providerVersions() : iterable
             'installed_version' => '1.0.0',
             'expected' => ['malukenho/zend-framework' => '1.0.0'],
         ];
+
+        yield 'version with leading "v" char but version prefixes are disabled via options' => [
+            'package' => 'malukenho/docheader',
+            'required_version' => '^1.0',
+            'installed_version' => 'v1.0.1',
+            'expected' => ['malukenho/docheader' => '^1.0.1'],
+            'options' => ['stripVersionPrefixes' => true],
```
Comment (Slamdunk) [inline test/BumpIntoTest.php:169]: We should add this to readme somehow

**Review by Slamdunk — CHANGES_REQUESTED** [review]

**Comment by malukenho** [comment]
I'm all fine with doing multiples SMALL changes in a single PR if they are committed separately, so it is okay.

It looks nice! We just need to address [this](https://github.com/malukenho/mcbumpface/pull/25#discussion_r370976506) comment and merge it 👍

**Comment by boesing** [comment]
I've added configuration section to the README file

https://github.com/boesing/mcbumpface/tree/feature/strip-version-prefixes#configuration-optional

**Review by Slamdunk — APPROVED** [review]

**Comment by malukenho** [comment]
Thank you very much @boesing 🎉

**Comment by Slamdunk** [comment]
@malukenho may we tag this with a new minor, like `1.1.0`?

**Comment by malukenho** [comment]
Ofc! I'll do it at the end of the day.

**Comment by Slamdunk** [comment]
Thank you very much :beers:

**Comment by malukenho** [comment]
Just FYI `1.1.0` released @Slamdunk 🎉

**Pull request merged**